tests/1 - AppScenario.js

```diff
@@ -0,0 +1,8 @@
+description: "The header should show metadata on the loaded tax benefit system.",
```

Comment:
Not sure if it's a good practise to put "should" all the time. Affirmatives sentences are less heavy.

"The header displays metadata on the loaded tax and benefit system" ?
"Make sure the header displays metadata on the loaded tax and benefit system" ?

@MattiSG, what is your watai expert opinion ?